test/integration/connect/envoy/run-tests.sh

```diff
@@ -43,6 +43,16 @@ function network_snippet {
     echo "--net container:envoy_consul-${DC}_1"
 }
 
+function aws_creds_snippet {
+  local snippet=""
+  if [[ -n "$AWS_ACCESS_KEY_ID" && -n "$AWS_SECRET_ACCESS_KEY" ]]; then
+    snippet="-e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY"
+    # This allows the Lambda integration tests to run when AWS credentials are obtained through STS
+    [[ -n "$AWS_SESSION_TOKEN" ]] && snippet="${snippet} -e AWS_SESSION_TOKEN=$AWS_SESSION_TOKEN"
```

Comment:
The session token is required for our local testing because our AWS creds are provisioned via STS. I guess I could clarify the comment...

Reply:
Actually the session token is also required in CI.